### PR TITLE
feat(init) add new ´init´ command

### PR DESCRIPTION
**Pull request by johanhammar**
Add new command  to initialize a local workspace with a default cli.conf.yaml configuration file

### CODE REVIEW

**Comment by johanhammar** [comment]
I used this when starting development on our portal. Might be useful for others as well.

File: src/commands/init.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+# Kong Admin Token
+# @optional
+kong_admin_token: ''`
+
+export default async (args): Promise<void> => {
+
+  let path = join("workspaces", args.workspace)
+
+  await fs.mkdir(path, { recursive: true }, (err) => {
```
Comment (Mierenga) [inline src/commands/init.ts:18]: Looking at the [`fs-extra` docs](https://github.com/jprichardson/node-fs-extra#sync-vs-async-vs-asyncawait), `fs.mkdir` only returns a promise if a callback is not passed. So I think this should just be `await fs.mkdir(path, { recursive: true })`, or else there will be no result to `await`. And then any error will be thrown automatically.

**Comment by Mierenga** [comment]
Hi @johanhammar, thanks for this PR as well. This is a handy addition to the CLI. I left one comment in the code about usage of `fs-extra`. Other than that, if you could adjust the few style differences with this repo's eslint config and rebase with `master`, we'll be able to merge this in.

**Comment by johanhammar** [comment]
Hi @Mierenga 

I've rebased to master, fixed the eslint warnings and your comments. Let me know if there anything else I can or should fix. Thanks

**Review by Mierenga — APPROVED** [review]
Looks great, @johanhammar, thanks for your work on this! Feel free to merge this in and we will add it to the next release.

**Comment by johanhammar** [comment]
Thanks :+1: 

I don't think I have the permissions to squash and merge the commits?

**Comment by Mierenga** [comment]
My mistake, I thought there were permissions configured for contributors to be able to merge after a PR was approved. I wanted to make sure the contribution got attributed to you. I merged it in and it shows you as the author :)

**Pull request merged**